src/main/java/com/cburch/logisim/gui/prefs/SimOptions.java

```diff
@@ -221,108 +220,106 @@ public SimOptions(PreferencesFrame window) {
-      add(kmap1ColorTitle, c);
-      c.gridx++;
-      kmap1Color = new ColorChooserButton(window, AppPreferences.KMAP1_COLOR);
-      add(kmap1Color, c);
-      c.gridx++;
-      add(kmap2ColorTitle, c);
-      c.gridx++;
-      kmap2Color = new ColorChooserButton(window, AppPreferences.KMAP2_COLOR);
-      add(kmap2Color, c);
-
-      c.gridx = 0;
-      c.gridy++;
-      add(kmap3ColorTitle, c);
-      c.gridx++;
-      kmap3Color = new ColorChooserButton(window, AppPreferences.KMAP3_COLOR);
-      add(kmap3Color, c);
-      c.gridx++;
-      add(kmap4ColorTitle, c);
-      c.gridx++;
-      kmap4Color = new ColorChooserButton(window, AppPreferences.KMAP4_COLOR);
-      add(kmap4Color, c);
-
-      c.gridx = 0;
-      c.gridy++;
-      add(kmap5ColorTitle, c);
-      c.gridx++;
-      kmap5Color = new ColorChooserButton(window, AppPreferences.KMAP5_COLOR);
-      add(kmap5Color, c);
-      c.gridx++;
-      add(kmap6ColorTitle, c);
-      c.gridx++;
-      kmap6Color = new ColorChooserButton(window, AppPreferences.KMAP6_COLOR);
-      add(kmap6Color, c);
-
-      c.gridx = 0;
-      c.gridy++;
-      add(kmap7ColorTitle, c);
-      c.gridx++;
-      kmap7Color = new ColorChooserButton(window, AppPreferences.KMAP7_COLOR);
-      add(kmap7Color, c);
-      c.gridx++;
-      add(kmap8ColorTitle, c);
-      c.gridx++;
-      kmap8Color = new ColorChooserButton(window, AppPreferences.KMAP8_COLOR);
-      add(kmap8Color, c);
-
-      c.gridx = 0;
-      c.gridy++;
-      add(kmap9ColorTitle, c);
-      c.gridx++;
-      kmap9Color = new ColorChooserButton(window, AppPreferences.KMAP9_COLOR);
-      add(kmap9Color, c);
-      c.gridx++;
-      add(kmap10ColorTitle, c);
-      c.gridx++;
-      kmap10Color = new ColorChooserButton(window, AppPreferences.KMAP10_COLOR);
-      add(kmap10Color, c);
-
-      c.gridx = 0;
-      c.gridy++;
-      add(kmap11ColorTitle, c);
-      c.gridx++;
-      kmap11Color = new ColorChooserButton(window, AppPreferences.KMAP11_COLOR);
-      add(kmap11Color, c);
-      c.gridx++;
-      add(kmap12ColorTitle, c);
-      c.gridx++;
-      kmap12Color = new ColorChooserButton(window, AppPreferences.KMAP12_COLOR);
-      add(kmap12Color, c);
-
-      c.gridx = 0;
-      c.gridy++;
-      add(kmap13ColorTitle, c);
-      c.gridx++;
-      kmap13Color = new ColorChooserButton(window, AppPreferences.KMAP13_COLOR);
-      add(kmap13Color, c);
-      c.gridx++;
-      add(kmap14ColorTitle, c);
-      c.gridx++;
-      kmap14Color = new ColorChooserButton(window, AppPreferences.KMAP14_COLOR);
-      add(kmap14Color, c);
-
-      c.gridx = 0;
-      c.gridy++;
-      add(kmap15ColorTitle, c);
-      c.gridx++;
-      kmap15Color = new ColorChooserButton(window, AppPreferences.KMAP15_COLOR);
-      add(kmap15Color, c);
-      c.gridx++;
-      add(kmap16ColorTitle, c);
-      c.gridx++;
-      kmap16Color = new ColorChooserButton(window, AppPreferences.KMAP16_COLOR);
-      add(kmap16Color, c);
-    }
+    c.gridx = 0;
```

Comment:
Can you please add `FIXME: refactor this code `  comment there?

Reply:
done